routemaster/client.rb

```diff
@@ -92,7 +92,7 @@ def _assert_valid_url(url)
     end
 
     def _assert_valid_topic(topic)
-      _assert (topic =~ /^[a-z_]{1,32}$/), 'bad topic name'
+      _assert (topic =~ /^[a-z_]+$/), 'bad topic name: must only include letters and underscores'
```

Comment:
may be still limit it to 64, although arbitrary length, but I can't think of a situation (until someone really comes up with one) where you will have a topic name longer than that.